modules/rosa-sts-creating-roles.adoc

```diff
@@ -0,0 +1,624 @@
+[id="rosa-sts-creating-roles_{context}"]
+= Creating IAM roles and policies for STS in ROSA
+
+Complete the following steps to set up IAM and OIDC access-based roles before creating your cluster.
+
+.Prerequisites
+
+* Review and complete the deployment prerequisites and policies.
+* Complete setting up the environment for STS.
+
+== Installer access role and policy
+This role is used by the {product-title} (ROSA) CLI to manage installation and deletion of ROSA clusters.
+
+.Procedure
+. Write the trust policy for the `ManagedOpenShift-IAM-Role` installer access role to the [filename]`ManagedOpenShift_IAM_Role.json` file.
++
+.Command to write the trust policy
+[%collapsible]
+====
+[source,terminal]
+----
+$ cat << EOM > ManagedOpenShift_IAM_Role.json
```

Comment:
User have to delete the "$" to make the command run after they copy the command. It's not convenient, can this be fixed?